### ISSUE TITLE
No automatic hot-end cooling after reset

### ISSUE THREAD

**Issue by Highcooley** [repetier/Repetier-Firmware#69]
When I have to reset my printer for what ever reason and the hot-end is still above 50°C, the automated cooling stops. Only after switching the hot-end back on and off again, the cooling resumes. It would be nice to have the firmware check hot-end temperatures on startup and resume cooling if necessary. THX
